### DISCUSSION TITLE
Can't Submit Application for Student Developer Pack

### DISCUSSION THREAD

**Discussion by KunalKapoor25** [community/community#113234] [GitHub Education]
### Hi everyone,

After uploading proof of my academic status and clicking "Process my application", the page loads for a few seconds but it doesn't indicate that my application has actually gone through. I've received no email/communication indicating that my application was even submitted; I first tried to apply for the Student Developer Pack a month ago. I've tried in both Chrome and Safari.

**Comment by lin20260407** [comment]
It sounds like you're encountering issues with submitting your application for the Student Developer Pack. Here are some steps you can take to troubleshoot and resolve the problem:

Check Your Email Spam Folder: Sometimes, confirmation emails might be redirected to your spam or junk folder. Make sure to check these folders for any communication regarding your application.

Try a Different Browser: Although you've already tried Chrome and Safari, consider using a different browser or even a private/incognito window to see if the issue persists. Sometimes browser extensions or cookies can interfere with the submission process.

Clear Browser Cache and Cookies: Clearing your browser's cache and cookies can resolve issues related to webpage loading and form submissions. After clearing them, try submitting your application again.

Disable Browser Extensions: Browser extensions, particularly those related to ad-blocking or security, might prevent forms from submitting correctly. Try disabling these extensions temporarily and then attempt to submit your application again.

Wish you have a nice day
